install-lightup.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+source user_config.sh
+source fixed_config.sh
+
+set -e
+set -x
+
+APP_NAME=lightup
+LICENSE_FILE=./license/${LIGHTUP_CUSTOMER_TLA}.yaml
+SHARED_PASSWORD=changeImmediately123
```

Comment:
I would really like all of these to be part of the token so, there is nothing in the code (if at all possible).

Reply:
I guess you are not wanting additional dependencies and want to run this as pure bash. Makes sense then.

Reply:
as discussed, ignoring for now.